### ISSUE TITLE
Optional building kernel

### ISSUE THREAD

**Issue by sijo-mathew** [avionic-design/linux-l4t#4]
Now i am trying to update the kernel of Linux 4 tegra kernel with the Avionic kernel. But when i trued to build the kernel it is giving the error

$ sudo make O=build Tamonten-t30-tec_deconfig
  HOSTCC  scripts/basic/fixdep
  GEN     /media/Data/L4TBackup/kernel/kernel_source_ad/kernel_source/build/Makefile
make[2]: **\* No rule to make target `Tamonten-t30-tec_deconfig'.  Stop.
make[1]: **\* [Tamonten-t30-tec_deconfig] Error 2
make: **\* [sub-make] Error 2

I am following the steps given in TamontenNG_EvalKit_UserGuide.pdf
ie, Section 6.2.2 Install "Linux for Tegra", 8th Step

Basically I want to replace the kernel modules coming with https://developer.nvidia.com/linux-tegra driver with Avionic Kernel in the Ubuntu sample File system. So please help me in setting up Ubuntu file system with Avionic Kernel

**Comment by soerengrunewald** [comment]
Actually there is no such defconfig within this kernel tree. You have to use tegra_tec_ng_defconfig  instead.
Looks like a mismatch between documentation and the actual kernel sources at this point.
